sklearn/inspection/_plot/decision_boundary.py

```diff
@@ -355,8 +369,13 @@ def from_estimator(
             if is_regressor(estimator):
                 raise ValueError("Multi-output regressors are not supported")
 
-            # TODO: Support pos_label
-            response = response[:, 1]
+            # For the multiclass case, `_get_response_values` returns the response
+            # as-is. Thus, we have a column per class and we need to select the column
+            # corresponding to the positive class.
+            if pos_label is None:
+                pos_label = estimator.classes_[1]
```

Comment:
Actually we cannot reach this reach this part of the code because we would have already raise an error before.